apps/inbox/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from . import views
+
+app_name = 'inbox'
```

Comment:
that thing conflicts with app_name in apis, maybe its best to rename the URLs in api.inbox to inbox_api, and any occurrences if any (not sure)